src/wp-content/themes/twentytwentyone/assets/css/style-editor.css

```diff
@@ -2145,6 +2145,7 @@ pre.wp-block-verse {
 
 :root .is-small-text,
 :root .has-small-font-size {
+	--wp--preset--font-size--small: var(--global--font-size-sm);
```

Comment:
I wonder if now that we have variables --wp--preset--font-size--small if it still makes sense to have --global--font-size-sm? Maybe we could remove all --global--font-size variables and use the presets.

Reply:
I wanted to scope down this to the minimal patch that works. Didn't want to rebuild how the theme is built to avoid falling into deep holes.